packages/i18n/src/locales/en_US.ts

```diff
@@ -1134,7 +1134,7 @@ const translations: Catalog = {
             },
             currentOp: {
               link: 'https://docs.mongodb.com/manual/reference/method/db.currentOp',
-              description: 'Calls the currentOp command. Returns a document that contains information on in-progress operations for the database instance. The db.currentOp() method wraps the database command currentOp.',
+              description: 'Runs an aggregation using $currentOp operator. Returns an aggregated document that contains information on in-progress operations for the database instance. For further information, see $currentOp.',
```

Comment:
The return value isn’t really aggregated (in the English sense of the word), right? I think “Returns a document” was good wording here

Reply:
I think I was more focused on the process but you're right the return value itself is not. Will change that.